### PR TITLE
Added some logs

### PR DESCRIPTION
**Pull request by AlessandroColucci**
Added some logs in order to make the execution more clear

### CODE REVIEW

**Review by nttdata-rtorsoli — DISMISSED** [review]

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/InteropProbingApplication.java [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+@Slf4j
 public class InteropProbingApplication {
 
   public static void main(String[] args) throws IOException {
+    log.info("Eservice-Registry-Updater started at: {}", LocalDateTime.now(ZoneOffset.UTC));
     Injector injector =
         Guice.createInjector(new BaseModule(), new PropertiesLoader(), new SqsConfig());
     ServicesReceiver servicesReceiver = injector.getInstance(ServicesReceiver.class);
     servicesReceiver.receiveStringMessage();
+    log.info("Eservice-Registry-Updater started at: {}", LocalDateTime.now(ZoneOffset.UTC));
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/eservice/registry/updater/InteropProbingApplication.java:23]:
```suggestion
    log.info("Eservice-Registry-Updater ended at: {}", LocalDateTime.now(ZoneOffset.UTC));
```

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/consumer/ServicesReceiver.java [hunk 2]
```diff
@@ -87,6 +89,7 @@ public void receiveStringMessage() throws IOException {
         AWSXRay.endSegment();
       }
       sqsMessages = sqs.receiveMessage(receiveMessageRequest).getMessages();
+      logger.logNumberMessagesReceived(Objects.nonNull(sqsMessages) ? sqsMessages.size() : 0);
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/eservice/registry/updater/consumer/ServicesReceiver.java:92]: may you print two different messages when logging before and after the loop?

Comment (AlessandroColucci) [reply src/main/java/it/pagopa/interop/probing/eservice/registry/updater/consumer/ServicesReceiver.java:92]: It is the exact same operation. We just repeat it to iterate over the messages.

Comment (galales) [reply src/main/java/it/pagopa/interop/probing/eservice/registry/updater/consumer/ServicesReceiver.java:92]: you're right, I misread the code block 👍

**Review by nttdata-rtorsoli — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Pull request merged**